### ISSUE TITLE
Static Message Translations issues

### ISSUE THREAD

**Issue by bavodeclercq** [spicywebau/craft-neo#619]
#### Description

1. Static Message Translations are translated in the settings view, which results in overwritten blocktype titles on save.
2. Using site.php instead of neo.php for content translations would make more sense (cf. Matrix fields). 
I'm using the following workaround to "fix" this:

```php
<?php
use craft\i18n\PhpMessageSource;

return PhpMessageSource::loadMessages('site', 'en');
```

#### Steps to reproduce
1. Create a Neo field
2. Use a Static Message Translation as blocktype title (for example: 'translate.blockType')
3. Translate the Static Message Translation via site.php (when using my workaround) or neo.php
4. Edit Neo field and save it
5. The Static Message Translation is now replaced by the translation. 

#### Other information
- Neo version: 2.13.12
- Craft version: Craft Pro 3.7.46

**Comment by ttempleton** [comment]
Please provide detailed steps to reproduce.

**Comment by bavodeclercq** [comment]
> Please provide detailed steps to reproduce.

My bad, added it to the original post.

**Comment by ttempleton** [comment]
Thanks for that, I was able to reproduce it and it should now be fixed in Neo 2.13.13.

It turned out to also affect Neo on Craft 4 as well (with the difference that it does use site.php instead of neo.php on Craft 4) so that will also be fixed in the next Neo 3 release.
